docs/excel/custom-functions-json.md

```diff
@@ -130,9 +130,8 @@ The `options` object enables you to customize some aspects of how and when Excel
 
 |  Property  |  Data type  |  Required  |  Description  |
 |:-----|:-----|:-----|:-----|
-|  `cancelable`  |  boolean  |  No<br/><br/>Default value is `false`.  |  If `true`, Excel calls the `onCanceled` handler whenever the user takes an action that has the effect of canceling the function; for example, manually triggering recalculation or editing a cell that is referenced by the function. If you use this option, Excel will call the JavaScript function with an additional `caller` parameter. (Do ***not*** register this parameter in the `parameters` property). In the body of the function, a handler must be assigned to the `caller.onCanceled` member. For more information, see [Canceling a function](custom-functions-web-reqs.md#canceling-a-function). |
-|  `requiresAddress`  | boolean | No <br/><br/>Default value is `false`. | <br /><br /> If true, your custom function can access the address of the cell that invoked your custom function. To get the address of the cell that invoked your custom function, use context.address in your custom function. For more information, see [Determine which cell invoked your custom function](/office/dev/add-ins/excel/custom-functions-overview#determine-which-cell-invoked-your-custom-function). Custom functions cannot be set as both streaming and requiresAddress. When using this option, the 'invocationContext' parameter must be the last parameter passed in options. |
-|  `stream`  |  boolean  |  No<br/><br/>Default value is `false`.  |  If `true`, the function can output repeatedly to the cell even when invoked only once. This option is useful for rapidly-changing data sources, such as a stock price. If you use this option, Excel will call the JavaScript function with an additional `caller` parameter. (Do ***not*** register this parameter in the `parameters` property). The function should have no `return` statement. Instead, the result value is passed as the argument of the `caller.setResult` callback method. For more information, see [Streaming functions](custom-functions-web-reqs.md#streaming-functions). |
+|  `cancelable`  |  boolean  |  No<br/><br/>Default value is `false`.  |  A cancelable function must declare an `invocation` parameter as the last parameter in the body of the function. If `"cancelable": true`, Excel calls the `invocation.onCanceled` method which you specify in the body of the function. (Do ***not*** register the `invocation` parameter in the `parameters` property). For more information, see [Streaming and cancelable functions](custom-functions-web-reqs.md#streaming-and-cancelable-functions). |
```

Comment:
I don't think sentence three needs to be in parentheses. Alternatively, consider taking that line and the repeated line in the next row and turning them into an !IMPORTANT section.